src/Impl/GuzzleHttpRequester.php

```diff
@@ -24,18 +24,33 @@ public function __construct(?ClientInterface $httpClient = null)
         $this->httpClient = $httpClient ?? new GuzzleHttpClient();
     }
 
-    public function executePost(
+    public function executeIngestionRequest(
```

Comment:
The name was wrong - `HttpRequester::executePost` sounded like executing any generic POST request.